### PR TITLE
[13.0][MIG] stock_quant_package_dimension

### PR DESCRIPTION
**Pull request by grindtildeath**
Forward-port of #590 

Depends on:
 - [x] #607 
 - [x] https://github.com/OCA/product-attribute/pull/551

### CODE REVIEW

**Review by Tonow-c2c — COMMENTED** [review]
comment idem as https://github.com/OCA/product-attribute/pull/551/files#diff-7204ef8da825976a99b3672ada1c75c8

File: stock_quant_package_dimension/models/stock_quant_package.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        res = super().auto_assign_packaging()
+        return res
+
+    def write(self, vals):
+        res = super().write(vals)
+        if self.env.context.get("_auto_assign_packaging") and vals.get(
+            "product_packaging_id"
+        ):
+            for package in self:
+                package.onchange_product_packaging_id()
```
Comment (simahawk) [inline stock_quant_package_dimension/models/stock_quant_package.py:39]: can't we use a computed / writable field now?

**Review by simahawk — COMMENTED** [review]
LG overall

File: stock_quant_package_dimension/models/stock_quant_package.py (hunk 1 above)
Comment (simahawk) [reply stock_quant_package_dimension/models/stock_quant_package.py:39]: not a blocker. Add a TODO if you think is will suit here.

**Review by simahawk — APPROVED** [review]

**Comment by simahawk** [comment]
@grindtildeath pre-commit is not happy yet :)

**Review by jgrandguillaume — APPROVED** [review]
LGTM

**Pull request merged**